docs/deployment/00-prereqs.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kudo-system
```

Comment:
so moving KUDO from `system` to `kudo-system` ?

Reply:
yes